components/builder-worker/src/config.rs

```diff
@@ -68,6 +72,10 @@ impl Default for Config {
             bldr_channel: String::from("unstable"),
             bldr_url: url::default_bldr_url(),
             jobsrv: vec![JobSrvAddr::default()],
+            github_app_id: 5565,
+            github_private_key_path: PathBuf::from(
+                "/hab/svc/hab-jobsrv/files/github_app_private_key.pem",
```

Comment:
You probably want to set this default to `/hab/svc/builder-worker/files/github_app_private_key.pem` or just `/tmp`

Reply:
oops. thats what I meant to do